packages/mds-agency/tests/db-integration-tests.ts

```diff
@@ -57,7 +59,7 @@ function now(): Timestamp {
 
 const APP_JSON = 'application/json; charset=utf-8'
 
-const LA_CITY_BOUNDARY = '1f943d59-ccc9-4d91-b6e2-0c5e771cbc49'
+const LA_CITY_BOUNDARY_ID = '1f943d59-ccc9-4d91-b6e2-0c5e771cbc49'
```

Comment:
Seems like for testing purposes, these IDs and such could be city agnostic.

Reply:
This is the same UUID we use for the greater CoLA service area, so it's not really city agnostic at this point. Once we fully deprecate service areas and come up with some generic geospatial fixtures, this should be city agnostic.